test/deadlock_program/producer_consumer_spurious.c

```diff
@@ -31,16 +31,16 @@ do_consume()
   pthread_mutex_unlock(&mut);
 }
 
-void
-consumer(void)
+void *
+consumer()
```

Comment:
Should this function take a `void*` as an argument?

Reply:
Done.  Thanks.